src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/block/iscsi/iscsi.component.ts

```diff
@@ -93,18 +93,6 @@ export class IscsiComponent implements OnInit, OnDestroy {
 
   }
 
-  ngOnInit() {
-    this.refresh();
-
-    this.interval = setInterval(() => {
```

Comment:
Please remove the `interval` field from this component (line 21)

Reply:
done